spec/javascripts/components/story/expanded_story/expanded_story_state_spec.js

```diff
@@ -21,12 +22,64 @@ describe('<ExpandedStoryState />', () => {
   });
 
   describe("states at select", () => {
+    describe('when is estimated feature', () => {
+      const onEditSpy = sinon.spy();
```

Comment:
Let's avoid sharing a spy between the tests, if we leave it right here both tests will use the same instance of the spy and it may lead to unwanted behavior. Can we move it to inside the `beforeEach` as well?